airbyte-integrations/connectors/source-zendesk-sunshine/acceptance-test-config.yml

```diff
@@ -0,0 +1,24 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/source-acceptance-tests.md)
+# for more information about how to configure these tests
+connector_image: airbyte/source-zendesk-sunshine:dev
+tests:
+  spec:
+    - spec_path: "source_zendesk_sunshine/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      validate_output_from_all_streams: yes
+#  incremental: #  complex state ( {parent_id: {cur_field: value}} still not supported )
```

Comment:
should this be deleted?

Reply:
@davinchia ideally we want to have all tests enabled, this comment explains why this particular test has been disabled, WDYT?

Reply:
it's okay to leave it. I think we should create a github issue so we don't forget to follow up with this.